server/public/model/insights.go

```diff
@@ -268,6 +268,7 @@ func StartOfDayForTimeRange(timeRange string, location *time.Location) *time.Tim
 // GetStartOfDayForTimeRange gets the unix start time in milliseconds from the given time range.
 // Time range can be one of: "today", "7_day", or "28_day".
 func GetStartOfDayForTimeRange(timeRange string, location *time.Location) (*time.Time, *AppError) {
```

Comment:
Shouldn't we remove this whole model file? Wondering why Harshill didn't remove it